pkg/crc/preflight/preflight_linux.go

```diff
@@ -295,3 +282,10 @@ func distro() *linux.OsRelease {
 	}
 	return distro
 }
+
+func resolvectlExists() bool {
+	if _, err := os.Stat("/usr/bin/resolvectl"); err != nil {
+		return false
+	}
+	return true
```

Comment:
This could be `return crcos.FileExists("/usr/bin/resolvectl")`

Reply:
Would it be better to replace the `resolvectlExists()` function and calls with that function completely?

Reply:
yep, makes sense, avoids the 1 line helper. Add a `const resolvectlPath="/usr/bin/resolvectl"`  though if you use `crcos.FileExists()` in the other code.

Reply:
I have that implemented locally, but added that const to the linux_constants file. Would you rather that be done within the scope of the preflight? Wasn’t sure if that was the appropriate place for such a variable.

Reply:
This is no longer relevant with the latest commit pushed.